src/Mapping/Formatter/BooleanFormatter.php

```diff
@@ -35,9 +36,9 @@ public function bind(string $key, Data $data) : BindResult
     public function unbind(string $key, $value) : Data
     {
         if (!is_bool($value)) {
-            // @todo throw exception
+            throw new InvalidValue(sprintf('Value of "%s" must be a Boolean', $key));
```

Comment:
Should be in the formatter exception namesapce and be fromNonBoolean(), but I gonna do that myself after merge.